lib/package.py

```diff
@@ -56,7 +56,8 @@ def __init__(self, name):
         self.download_source = None
         self.install_dependencies = []
         self.build_dependencies = []
-        self.result_packages = []
+        self.artifacts_dir = os.path.join(
```

Comment:
pls be consistent in the terminology, use either "artifacts" or "results". I prefer the latter, as it is a more universally know term. "artifacts" is Jenkins terminology.